conf/build.manifest

```diff
@@ -85,6 +85,8 @@ base-packages:
 - squashfs-tools
 - sysstat
 - truenas
```

Comment:
I think we should just change all of these packages to be prefixed with `name:` to keep it clean and concise. It will also allow for future flexibility if we ever have to add more options for each package.